jenkins-apb/README.md

```diff
@@ -0,0 +1,28 @@
+jenkins-apb
+======================
+
+## What it does
+This APB deploys a Jenkins server capable of managing OpenShift Pipeline builds and supporting OpenShift-based
+oauth login.
+
+## Requirements
+
+## Parameters
+* persistent: Optional, if true use a persistent volume for Jenkins data.
```

Comment:
@jcpowermac totally fine to include in another PR. We are still implementing plans in APBs so i don't expect that to be well documented yet.